Automattic/jetpack
# Build: Remove explicit Babel dependencies

#### Changes proposed in this Pull Request:

Per #11949, Jetpack relies on `@automattic/calypso-build` for all its Babel transforms. There are no more direct mentions of any other individual Babel plugins or presets in Jetpack's codebase, so it's safe to remove those dependencies from `package.json`

#### Testing instructions:

```
yarn distclean && yarn
yarn build-extensions
```

Verify that blocks still build and work.

#### Proposed changelog entry for your changes:

N/A
